src/script/scripting_client.cpp

```diff
@@ -58,6 +59,9 @@ ClientScripting::ClientScripting(Client *client):
 	lua_pushstring(L, "client");
 	lua_setglobal(L, "INIT");
 
+	lua_pushstring(L, "/");
+	lua_setglobal(L, "DIR_DELIM");
+
```

Comment:
That's what the client does. This is just so that builtin doesn't need extra code.